This patch fixes two things:
config.guess:
	i[34]86-*-FreeBSD* -> i386--bsd

libio/gen-params:
	We use hex in limits.h

Poul-Henning Kamp
phk@login.dkuug.dk

diff -C2 -r libg++-2.6.orig/config.guess libg++-2.6/config.guess
*** libg++-2.6.orig/config.guess	Sat Jul 16 12:11:08 1994
--- libg++-2.6/config.guess	Sun Jul 31 20:45:59 1994
***************
*** 197,201 ****
  	exit 0 ;;
      i[34]86:FreeBSD:*:*)
! 	echo ${UNAME_MACHINE}-unknown-freebsd${UNAME_RELEASE}
  	exit 0 ;;
      i[34]86:NetBSD:*:*)
--- 197,201 ----
  	exit 0 ;;
      i[34]86:FreeBSD:*:*)
! 	echo ${UNAME_MACHINE}-unknown-bsd
  	exit 0 ;;
      i[34]86:NetBSD:*:*)
diff -C2 -r libg++-2.6.orig/libio/gen-params libg++-2.6/libio/gen-params
*** libg++-2.6.orig/libio/gen-params	Mon Jul 18 18:04:09 1994
--- libg++-2.6/libio/gen-params	Sun Jul 31 20:50:12 1994
***************
*** 415,432 ****
  default_int64=
  INT16=32767
  INT32=2147483647
  INT64=9223372036854775807
  
! if [ "${SHRT_MAX}" = $INT16 ] ; then
    default_int16=short
!   if [ "${LONG_MAX}" = $INT32 ] ; then
      default_int32=long
!     if [ "${LONG_LONG_MAX}" = $INT64 ] ; then
        # Most Unices, DOS
        default_int64="long long"
      fi
!   elif [ "${INT_MAX}" = $INT32 ] ; then
      default_int32=int
!     if [ "${LONG_MAX}" = $INT64 ] ; then
        # Some 64-bit Unices, like OSF/1 on the Alpha
        default_int64=long
--- 415,435 ----
  default_int64=
  INT16=32767
+ INT16X=0x7fff
  INT32=2147483647
+ INT32X=0x7fffffff
  INT64=9223372036854775807
+ INT64X=0x7fffffffffffffff
  
! if [ "${SHRT_MAX}" = $INT16 -o "${SHRT_MAX}" = $INT16X ] ; then
    default_int16=short
!   if [ "${LONG_MAX}" = $INT32  -o "${LONG_MAX}" = $INT32X] ; then
      default_int32=long
!     if [ "${LONG_LONG_MAX}" = $INT64  -o "${LONG_LONG_MAX}" = $INT64X] ; then
        # Most Unices, DOS
        default_int64="long long"
      fi
!   elif [ "${INT_MAX}" = $INT32  -o "${INT_MAX}" = $INT32X] ; then
      default_int32=int
!     if [ "${LONG_MAX}" = $INT64  -o "${LONG_MAX}" = $INT64X] ; then
        # Some 64-bit Unices, like OSF/1 on the Alpha
        default_int64=long
